src/Indice.Features.Risk.Core/Stores/RiskEventStoreEntityFrameworkCore.cs

```diff
@@ -27,4 +29,84 @@ internal class RiskEventStoreEntityFrameworkCore : IRiskEventStore
         }
         return await query.ToListAsync();
     }
+
+    public async Task<ResultSet<RiskEvent>> GetList(ListOptions<AdminRiskFilter> options) {
+        var query = _dbContext.RiskEvents.AsNoTracking().AsQueryable();
+        query = ApplyFilter(query, options.Filter.Filter);
+        return await query.ToResultSetAsync(options);
+    }
+
+    private IQueryable<RiskEvent> ApplyFilter(IQueryable<RiskEvent> query, FilterClause[] filter) {
+        foreach (var clause in filter) {
+            if (string.IsNullOrWhiteSpace(clause.Member)) {
+                continue;
+            }
+
+            if (clause.Member.ToLower() == "from" && DateTimeOffset.TryParse(clause.Value, out var dateFrom)) {
+                query = query.Where(c => c.CreatedAt >= dateFrom.Date);
+            }
+
+            if (clause.Member.ToLower() == "to" && DateTimeOffset.TryParse(clause.Value, out var dateTo)) {
+                query = query.Where(c => c.CreatedAt <= dateTo.Date.AddDays(1));
```

Comment:
γιατί μια μέρα μέτα το dateTo;

Reply:
για να μπορει να επιστρεψει μεχρι και ολη την προηγουμενη ημερα, μεχρι την αρχη της επομενης.

Reply:
περιμένω βελτίωση εδώ, αν μιλάμε να πάει μέχρι πχ χθες το μεσημέρι, με τον τρόπο που το έχεις κάνει πάει μέχρι σήμερα το μεσημέρι, το οποίο δεν ειναι η σωστή συμπεριφορά.

Reply:
όχι φίλε, θα μας ζητήσει να μπει και ωρα κάποια στιγμή και δεν θα παίζει, ειναι ευθραυστος κώδικας και δεν μ'αρεσει. Κάνε κάτι απλό σε αυτά τα πλαίσια https://www.c-sharpcorner.com/blogs/datetime-start-and-end-of-day-extension-methods.